test/functional/tests/fault_injection/test_reboot.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import os
+import pytest
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from core.test_run import TestRun
+from storage_devices.disk import DiskTypeSet, DiskType, DiskTypeLowerThan
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_utils.filesystem.file import File
+from test_utils.size import Size, Unit
+
+
+mount_point = "/mnt/test"
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.parametrize("reboot_type", ["soft", "hard"])
+@pytest.mark.require_plugin("power_control")
+def test_planned_reboot(reboot_type, cache_mode):
+    with TestRun.step("Prepare CAS device."):
+        cache_dev = TestRun.disks['cache']
+        core_dev = TestRun.disks['core']
+        cache = casadm.start_cache(cache_dev, cache_mode)
+        core = cache.add_core(core_dev)
+        core.create_filesystem(Filesystem.ext4, blocksize=int(Size(1, Unit.Blocks4096).get_value()))
```

Comment:
You don't need to call `get_value()` method here. It's called automatically when casting object of `Size` to `int`.